src/js/helpers/periodPickerHelper.js

```diff
@@ -20,17 +20,41 @@ export const mostRecentPeriod = () => {
 
     let period = 12;
 
-    if (today.isBetween(moment(`01/01/${year}`, 'MM-DD-YYYY'), moment(`01-08-${year}`, 'MM-DD-YYYY'))) {
-        year -= 1;
+    if (today.isBetween(moment(`10/18/${year}`, 'MM-DD-YYYY'), moment(`12/05/${year}`, 'MM-DD-YYYY'))) {
```

Comment:
These should go through the day after the _end date_ of the reporting window (Dec 19, Jan 18, Feb 16, etc.)

Reply:
10/18 to 12/18: Period 12
12/19 to 1/17: Period 2
1/18 to 2/15: Period 3
2/16 to 3/18: Period 4...